packages/docs-page/index.tsx

```diff
@@ -32,7 +33,7 @@ interface DocsPageInnerProps {
   pageTitle: string
   baseRoute: string
   githubFileUrl: string
-  product: { name: string; slug: string }
+  product: { name: string; slug: Products }
```

Comment:
Since you're narrowing the type accepted here, we probably need a major version bump for the `react-docs-page` package.

Reply:
Changed to **major**!